internal/benchmarks/benchmarks_test.go

```diff
@@ -258,6 +258,7 @@ func BenchmarkAllGadgetsWithContainers(b *testing.B) {
 							ctx,
 							"",
 							runtime,
+							runtime.ParamDescs().ToParams(),
```

Comment:
What about squashing this commit with `pkg/runtime: add RuntimeParams (in addition to RuntimeGlobalParams) t…`?